#include "exception.h"

#include "util/except.h"

void exception_throw(void* exception_obj) {
    // TODO: this
    ASSERT(0);
}
